packages/dx-react-grid-export/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "@devexpress/dx-react-grid-export",
+  "version": "2.4.0",
+  "description": "Export component for DevExtreme Reactive Grid",
+  "author": {
+    "name": "Developer Express Inc.",
+    "url": "https://www.devexpress.com/"
+  },
+  "bugs": {
+    "url": "https://github.com/DevExpress/devextreme-reactive/issues"
+  },
+  "homepage": "https://devexpress.github.io/devextreme-reactive/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/DevExpress/devextreme-reactive.git"
+  },
+  "keywords": [
+    "react",
+    "data",
+    "grid",
+    "export",
+    "excel",
+    "react-component"
+  ],
+  "license": "SEE LICENSE IN README.md",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/dx-react-grid-export.umd.js",
+  "module": "dist/dx-react-grid-export.es.js",
+  "globalName": "DevExpress.DXReactGridExport",
+  "files": [
+    "dist"
+  ],
+  "tsdoc": {
+    "tsdocFlavor": "AEDoc"
+  },
+  "types": "dist/dx-react-grid-export.d.ts",
+  "typings": "dist/dx-react-grid-export.d.ts",
+  "scripts": {
+    "test": "jest",
+    "test:watch": "jest --watch --runInBand",
+    "test:coverage": "jest --coverage",
+    "build": "rollup -c rollup.config.js",
+    "build:watch": "rollup -c rollup.config.js -w",
+    "update-api": "api-extractor run",
+    "dts": "tsc -p tsconfig.dts.json && api-extractor run -c api-extractor.dts.json && node ./merge-dts.js",
+    "lint": "tslint -p tsconfig.lint.json",
+    "lint:fix": "yarn lint -- --fix"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.7.2",
+    "@babel/plugin-transform-runtime": "^7.6.2",
+    "@babel/preset-env": "^7.7.1",
+    "@babel/preset-react": "^7.7.0",
+    "@devexpress/dx-core": "2.4.0",
+    "@devexpress/dx-react-core": "2.4.0",
+    "@devexpress/dx-react-grid": "2.4.0",
+    "@devexpress/dx-testing": "2.4.0",
+    "babel-core": "^7.0.0-bridge.0",
+    "core-js": "^3.4.1",
+    "enzyme": "3.10.0",
+    "enzyme-adapter-react-16": "^1.15.1",
+    "enzyme-to-json": "^3.4.3",
+    "jest": "^24.9.0",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-test-renderer": "^16.12.0",
+    "rollup": "^1.27.2",
+    "rollup-plugin-license": "^0.12.1",
+    "rollup-plugin-replace": "^2.2.0",
+    "rollup-plugin-typescript2": "^0.25.3",
+    "ts-jest": "^24.1.0",
+    "tslint": "^5.20.1",
+    "tslint-config-airbnb": "^5.11.2",
+    "typescript": "3.7.4"
+  },
+  "dependencies": {
+    "@devexpress/dx-grid-core": "2.4.0",
```

Comment:
Why is it a dependency? Can we move it into `peer`?

Reply:
We use core packages as dependencies, because user shouldn't install them manually or import from them

Reply:
`@devexpress/dx-grid-core` will be installed with `@devexpress/dx-react-grid` package